playbooks/webapp/webapp.yaml

```diff
@@ -16,19 +16,19 @@
       ansible.builtin.import_tasks: tasks/{{ operation }}.yaml
 
 - name: Deploy resource from Bastion
-  hosts: bastion
```

Comment:
Shouldn't be bastion here?

Reply:
The Bastion host is included in the inventory in the deploy_flask_app role. As a result, this task will be performed on the localhost, and the appropriate delegation will be handled by the role.